v2/tools/generator/internal/codegen/pipeline/resource_registration_file.go

```diff
@@ -132,6 +143,8 @@ func (r *ResourceRegistrationFile) generateImports() *astmodel.PackageImportSet
 
 	// We require these imports
 	clientGoSchemeImport := astmodel.NewPackageImport(astmodel.ClientGoSchemeReference).WithName("clientgoscheme")
+	genRuntimeReferenceImport := astmodel.NewPackageImport(astmodel.GenRuntimeReference)
```

Comment:
minor, style: I'd just do these 2 lines all in 1 like the other lines above/below are doing